tensorboard/plugins/hparams/tf_hparams_scatter_plot_matrix_plot/tf-hparams-scatter-plot-matrix-plot.html

```diff
@@ -152,10 +187,8 @@
               .attr("width", this.width)
               .attr("height", this.height)
 
-          // Delete all elements in the svg subtree and remove any
-          // selected group
+          // Delete all elements in the svg subtree
```

Comment:
No AI required: wish, instead, it updated via d3 these svg groups correctly :(

Reply:
Yes I agree, that would be more efficient. There are two important things I'd like to do after this gets released: 1. Refactor this and the scatter-plot large 'redraw' methods, and 2. Add integration tests that test the interaction of these elements. Probably better to do 2 before 1. As a related note, I'd be happy to get input on how you think is the best way to test these UI-interactive widgets. Let's talk about it in person.